internal/exec/engine.go

```diff
@@ -121,7 +121,12 @@ func (e *Engine) acquireConfig() (cfg types.Config, err error) {
 	// Create a new http client and fetcher with the timeouts set via the flags,
 	// since we don't have a config with timeout values we can use
 	timeout := int(e.FetchTimeout.Seconds())
-	err = e.Fetcher.UpdateHttpTimeoutsAndCAs(types.Timeouts{HTTPTotal: &timeout}, nil)
+	emptyProxy := types.Proxy{
```

Comment:
This can just be `types.Proxy{}` since you can use `range` over a `nil` list.

Reply:
Change addressed in latest rebase.